pkg/cmd/cli/cli.go

```diff
@@ -4,10 +4,10 @@ import (
 	"fmt"
 	"github.com/openshift/origin/pkg/cmd/flagtypes"
 	"github.com/openshift/origin/pkg/cmd/templates"
-	cmdutil "github.com/openshift/origin/pkg/cmd/util"
 	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
 	"github.com/spf13/cobra"
 	"io"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
```

Comment:
the RequireNoArguments() function seemed to move in the 1.4.1 release to the kube package, not sure why.